recipes/recipes-model.js

```diff
@@ -0,0 +1,25 @@
+const db = require('../data/dbConfig.js');
+
+module.exports = {
+  getRecipes,
+  getRecipe,
+  addRecipe
+};
+
+function getRecipes() {
+  return db('recipes');
+}
+
+function getRecipe(id) {
+  return db('recipes')
+    .where({id}, 'id')
+    .first();
+}
+
+function addRecipe(recipe) {
+  return db('recipes')
+    .insert(recipe)
+    .then(([id]) => {
+      return getRecipe(id);
```

Comment:
getRecipe is a asynchronous function. You should put another .then to handle the promise